pinot-common/pom.xml

```diff
@@ -96,6 +96,10 @@
       <groupId>org.antlr</groupId>
       <artifactId>antlr4-runtime</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.calcite</groupId>
+      <artifactId>calcite-core</artifactId>
```

Comment:
Does this have a lot more functionality than the complier that we won't need? If so, is there a different artifact to include that just has what we need?

Reply:
this is the artifact we need in order to use SqlParser(`https://github.com/apache/calcite/blob/master/core/src/main/java/org/apache/calcite/sql/parser/SqlParser.java`).

Reply:
So are we bringing in a lot more (in calcite-core) than we need?

Reply:
yes, we may exclude the unnecessary dependencies once conflict is found.